commands/entity.go

```diff
@@ -34,7 +35,11 @@ func (cmd *Entity) Run(scope scope.Scope, c *cli.Context) {
 	if c.Args().Present() {
 		cmd.show(scope.Application, c.Args().First())
 	} else {
-		cmd.list(scope.Application)
+		if scope.Entity == "" {
```

Comment:
This should never be true; scope.Entity is defaulted to be equal to scope.Application if you do not explicitly supply an entity scope.